$OpenBSD: patch-Makefile,v 1.1.1.1 2020/01/25 16:44:51 schwarze Exp $

In OpenBSD, we do not compile programs statically when it isn't necessary.

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -1,6 +1,5 @@
 VERSION=1.10.0
 CFLAGS+=-g -DVERSION='"$(VERSION)"' -Wall -Wextra -Werror -Wno-unused-parameter
-LDFLAGS+=-static
 INCLUDE+=-Iinclude
 PREFIX?=/usr/local
 _INSTDIR=$(DESTDIR)$(PREFIX)
